docs/fdc3-glossary.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Glossary of Terms
+sidebar_label: Glossary
+---
+
+For the purposes of this Standard, the following definitions apply. Other terms are defined when first used, at which place they appear in bold and italic type. Terms explicitly defined in this Standard are not to be presumed to refer implicitly to similar terms defined elsewhere. Terms not defined are assumed to be well-known in the financial services or software industries.
```

Comment:
Yes, @RexJaeschke suggested we capitalise3 Standard throughout, try not to use specification and generally to refer to 'this Standard' rather than the 'FDC3 Standard' (however I haven't scrubbed as many FDC3s as we could - I do find using it as a qualifier useful in places, particular when referring to one of the 'specifications'.

See https://github.com/finos/FDC3/issues/482 for his advice on this